cookbook/.kitchen.yml

```diff
@@ -0,0 +1,48 @@
+---
+driver:
+  name: vagrant
+  require_chef_omnibus: 11.16.4
+
+provisioner:
+  name: chef_zero
+  encrypted_data_bag_secret_key_path: <%= ENV['HOME'] %>/.chef/encrypted_data_bag_secret
+  client_rb:
+    environment: stage
```

Comment:
This seems to make some assumptions. Might it not be better to not specify an environment here?